modules/mysql/src/main/java/org/testcontainers/containers/MySQLContainer.java

```diff
@@ -54,6 +54,12 @@ public String getJdbcUrl() {
         return "jdbc:mysql://" + getContainerIpAddress() + ":" + getMappedPort(MYSQL_PORT) + "/" + databaseName;
     }
 
+    @Override
+    protected String constructUrlForConnection(String queryString) {
+        String separator = queryString.contains("?") ? "&" : "?";
+        return getJdbcUrl() + separator + "useSSL=false";
```

Comment:
You forgot the `queryString `. Probably should be `getJdbcUrl() +  queryString + separator + "useSSL=false"`. Or may be even
```
String url = super.constructUrlForConnection(queryString);
String separator = url.contains("?") ? "&" : "?";
return url + separator + "useSSL=false";
```